week-2-practise/level-3/4-slice.js

```diff
@@ -14,8 +24,8 @@ function replace(arr, index, value) {
 var numbers = [1, 3, 3];
 var names = ["Irina", "Ashleigh", "Mozafar"];
 
-var newNumbers = replace(arr, 1, 2);
-var newNames = replace(arr, 2, "Rares");
+var newNumbers = replace(numbers, 1, 2);
+var newNames = replace(names, 2, "Rares");
```

Comment:
This area is "DO NOT EDIT BELOW THIS LINE".
Can we do this without chancing this line?